### ISSUE TITLE
The GAN loss problem

### ISSUE THREAD

**Issue by Linfengscat** [YunYang1994/TensorFlow2.0-Examples#63]
The GAN loss binary_cross_entropy, I think, should take logits and labels both in (0,1),  Maybe the author forget to add sigmoid activation in the last dense layer of Discriminator? ? ?

**Comment by forever208** [comment]
following..
